### PR TITLE
Fix timeout problem in connect_pvlist()

### LINKED ISSUE #657: BUG: utils.connect_pvlist() does not connect
```py
In [6]: !caget ad:cam1:Acquire gp:UPTIME
ad:cam1:Acquire                Done
gp:UPTIME                      27 days, 21:37:52

In [7]: pvlist = ['ad:cam1:Acquire', 'gp:UPTIME']

In [8]: pvdict = connect_pvlist(pvlist)
Could not connect ad:cam1:Acquire
Could not connect gp:UPTIME
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-8-99345481b06f> in <module>
----> 1 pvdict = connect_pvlist(pvlist)

~/Documents/projects/BCDA-APS/apstools/apstools/utils/misc.py in connect_pvlist(pvlist, wait, timeout, poll_interval)
    561                     print(f"Could not connect {v.pvname}")
    562             if len(n) == 0:
--> 563                 raise RuntimeError("Could not connect any PVs in the list")
    564             obj_dict = n
    565 

RuntimeError: Could not connect any PVs in the list
```

### PR DESCRIPTION
**Pull request by prjemian**
- close #657

### CODE REVIEW

File: apstools/utils/misc.py [hunk 1]
```diff
@@ -546,7 +546,7 @@ def connect_pvlist(pvlist, wait=True, timeout=2, poll_interval=0.1):
         obj_dict[oname] = obj
 
     if wait:
-        times_up = time.time() + min(0, timeout)
+        times_up = time.time() + max(0, timeout)
         poll_interval = min(0.01, poll_interval)
```
Comment (hhhzzzsss) [inline apstools/utils/misc.py:550]: It's not directly related to the bug in question, but it looks like this should also be changed from min to max.

Comment (prjemian) [reply apstools/utils/misc.py:550]: Good catch!

**Comment by hhhzzzsss** [comment]
Everything else looks good to me

**Comment by prjemian** [comment]
@hhhzzzsss Thanks!

**Pull request merged**